docs/devconsole/README.md

```diff
@@ -0,0 +1,20 @@
+# GitOps Setup on DevConsole
+
+
+1. Setup a GitOps pipeline on your cluster
+    * Using ODO - https://gist.github.com/rohitkrai03/c01a93739dcfad8300d1e4e6450caba6
```

Comment:
Let's bring this in, rather than linking to a random (from a customer PoV) gist

We should ask Rohit to maintain it in here too, if there are changes.

Reply:
Included